### PR TITLE
Ported several /tg/ chems and balance changes

### PR DESCRIPTION
**Pull request by X-TheDark**
Special request by Adam Le-Tablemaster

Ported over chemistry additions and new descriptions from tgstation/-tg-station#13062 by @optimumtact
Oxadrolone ported from tgstation/-tg-station#11380 by @bgobandit
Haloperidol ported from tgstation/-tg-station#14057 by @markressler
### New toxins and healing chems: teslium, heparin, oxandrolone, haloperidol
- Teslium. Made from Plasma, Silver, and Black Powder, heated to 400K. Increases shock damage by 1.5, randomly shocks you for 5-20 (pre-adjustment) damage every 5-30 seconds. Metabolizes at a rate of 0.2 per tick. **Extremely deadly from me testing this.**
- Heparin. Made from Formaldehyde, Sodium, Chlorine, and Lithium. Makes you bleed aggressively and does 1 brute damage each tick. Metabolizes at a rate of 0.08 per tick. Sort of mediocre deadliness, but blood loss is relatively hard to counter.
- Oxandrolone. Made from 3 parts Carbon, 1 part of Hydrogen, Oxygen and Phenol. Twice as effective as silver sulfadiazine against heavy burns (>50 fire damage), half as effective otherwise.
- Haloperidol. Made from Chlorine, Fluorine, Aluminum, Potassium Iodide, and Oil. Quickly purges body of drugs (meth/bath salts/etc). Doesn't purge Stimulants (stimpacks), since those are medical drugs.
### Medical chem balances
- Synaptizine toxloss chance reduced to 30% (from 60%).
- Rezadone instantly heals all clone damage.
- Diphenhydramine drowsiness chance reduced to 10% (from 50%), removes 3 Histamine per tick (up from 1.5).
- Pentetic Acid no longer has a chance to cause brute damage.
- **Morphine nerfed, knocks you out after 24 cycles** (up from 12). After 12 cycles, you become drowsy.
- Oculine actually cures Blindness/Nearsightedness disabilities, not just eye damage.
- Atropine and Epinephrine now are 100% effective for anyone in crit (below 0 health).
### Toxin chem balances
- Amatoxin power increased from 1 to 2.5.
- Lexorin does not cause organ damage, but instead is even better at stopping breathing.

### CODE REVIEW

**Comment by FluffySurvivor** [comment]
MORPHINE NERF THANKS GOD

**Comment by ShadowDeath6** [comment]
So teslium will do 5-20 shock damage with or without the modifier it already applies to shock damage?

**Comment by X-TheDark** [comment]
> 5-20 (pre-adjustment)

So yeah, before the 1.5 multiplier, since it's applied during the shock damage application, so in reality, that's 7.5-35 damage you could get every 5-30 seconds.

**Comment by X-TheDark** [comment]
Working with a revenant has never been easier. Not only does his shocks increase in damage, but target gets randomly floored on it's own.

File: code/modules/reagents/Chemistry-Reagents/Medicine-Reagents.dm [hunk 1]
```diff
@@ -68,12 +68,12 @@
 /datum/reagent/medicine/adminordrazine/nanites
 	name = "Nanites"
 	id = "nanites"
-	description = "Tiny machines capable of rapid cellular regeneration."
+	description = "Tiny nanomachines capable of rapid cellular regeneration."
```
Comment (B1gb3ast) [inline code/modules/reagents/Chemistry-Reagents/Medicine-Reagents.dm:71]: Tiny nanomachines?
What the fuck dude.
Just make it nanomachines or tiny machines. Not Tiny nanomachines.

Comment (DavidEdwards) [inline code/modules/reagents/Chemistry-Reagents/Medicine-Reagents.dm:71]: Because it sounds awesome.

**Comment by Ephemeralis** [comment]
I am woefully late but "haloperidol" is totally the wrong name for this sort of drug, it is terrible and it needs changing immediately.

You've literally brought me out of retirement for this.

**Pull request merged**